dev/io.openliberty.org.eclipse.microprofile/faulttolerance.4.1.bnd

```diff
@@ -19,9 +19,9 @@ Export-Package: \
   org.eclipse.microprofile.faulttolerance.exceptions;version=1.0
 
 Include-Resource: \
-  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;4.0.2;EXACT}
+  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;4.1.0.RC1;EXACT}
```

Comment:
```suggestion
  @${repo;org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;4.1.RC1;EXACT}
```

Reply:
This is not correct. The version here is the OSGi version of the bundle which has been retrieved from maven. As it's an OSGi version, it must have three digits.

`4.1.0.RC1` is the version that BND shows in my eclispe in the repositories view. I'm not sure whether this has been generated by BND or retrieved from the `Bundle-Version` header.

We could probably refer to the jar here by its symbolic name rather than its maven groupId and artifactId if we wanted to.

With the suggested change, the build fails with
```
error  : ${repo} macro refers to an artifact org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api-4.1.RC1 (EXACT) that has an error: org.eclipse.microprofile.fault-tolerance:microprofile-fault-tolerance-api;version=4.1.RC1 Invalid version
```

Reply:
Oh yeah.  Saw this elsewhere with other things where the maven version doesn't match the bundle version.  So much fun.  Marking resolved.